client/src/components/FilmList.jsx

```diff
@@ -1,13 +1,17 @@
 import { useState, useEffect } from 'react'
-import * as ioicons from 'react-icons/io5'
 import ViewFilm from './ViewFilm'
-import CreateContact from './CreateContact'
+import Form from './Form'
```

Comment:
ESLint was bugging me about your file path here. Since the file name is lowercase, have its reference here be lowercase, too